.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] You have put `node_modules` here but still, you have pushed the node_modules. Please remove the node_modules folder from Github.